lib/thunder/bloc/thunder_bloc.dart

```diff
@@ -317,4 +339,31 @@ class ThunderBloc extends Bloc<ThunderEvent, ThunderState> {
   void _onFabSummonToggle(OnFabSummonToggle event, Emitter<ThunderState> emit) {
     emit(state.copyWith(isFabSummoned: !state.isFabSummoned));
   }
+
+  void _onAddAnonymousInstance(OnAddAnonymousInstance event, Emitter<ThunderState> emit) async {
+    final SharedPreferences prefs = (await UserPreferences.instance).sharedPreferences;
```

Comment:
Just curious, but should we include this when we export/import settings? My thought here is that maybe we should exclude these settings but let me know your thoughts!

Reply:
Yeah I agree. I added both anonymous instance settings to the import/export exclusion list.